app/models/card.rb

```diff
@@ -2,7 +2,7 @@ class Card < ActiveRecord::Base
 validates :original_text, :translated_text, presence: true, uniqueness: { case_sensitive: false }
 validates :review_date, presence: true
 validate :must_not_be_equal
-scope :expired, -> { where('review_date <= ?', DateTime.now) }
+scope :expired, -> { where('review_date <= ?', Date.today) }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.